.github/workflows/version.yml

```diff
@@ -14,21 +14,25 @@ jobs:
       - uses: actions/checkout@v2
         with:
           ssh-key: ${{ secrets.DEPLOY_KEY }} # with this we can bypass branch protection rules here (no-push)
+
       - uses: actions/setup-node@v2
         with:
-          node-version: "14"
-      - name: Bump version number
+          node-version: "16"
+
+      - name: Setup git
         run: |
           git config user.name "github-actions"
           git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
 
+      - name: Bump version number
+        run: |
           # this just prints the new version number - will not update any files
-          NEW_VERSION=$(node build/bump-version.js --print-version)
+          NEW_VERSION=$(build/version-next.sh)
 
           # auto-increment version number upon merging pull requests
-          node build/bump-version.js --apply
+          build/bump.sh --apply
 
           # create commit
-          git add -A
-          git commit -m "chore: bump version to v${NEW_VERSION}"
+          git add frontend/package.json backend/api/Pims.Api.csproj
```

Comment:
**Security:** Ensure we only add relevant files to the repo. We wouldn't want to add other files that might have been modified (by a 3rd party potentially)